src/style.css

```diff
@@ -1 +1,115 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  src: url('/src/fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('/src/fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: bold;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.search__container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  padding: 20px 8px;
+}
+
+.search-bar {
+  display: flex;
+  width: 100%;
+}
+
+.search-bar__input {
+  display: block;
+  width: 100%;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
```

Comment:
Consider adding a fallback font stack for better cross-browser compatibility. While Avenir is specified, including common system fonts like Arial, Helvetica, and sans-serif ensures the text is displayed correctly if the custom font fails to load.